packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1085,9 +1085,12 @@ class Auth {
             const errorData = await response.json()
             throw new Error(errorData.message || 'API validation failed')
         }
-        // Await 1s before calling authorizeIDP again
-        await new Promise((resolve) => setTimeout(resolve, 1000))
         if (onClient()) {
+            // To avoid seeing 409 Conflict error from SLAS,
+            // we need to wait a second before calling the url again
+            await new Promise((resolve) => setTimeout(resolve, 1000))
+
+            // Navigate to the url and follow its redirect this time
```

Comment:
Bonus: executing the 1-second delay only when it's really needed (when it's on the client side). Plus, tweaking the comment to make it clearer why we needed the delay.